tests/unit/specs/components/widget/WidgetProject.spec.js

```diff
@@ -0,0 +1,57 @@
+import { createLocalVue, shallowMount } from '@vue/test-utils'
+
+import { Core } from '@/core'
+import WidgetProject from '@/components/widget/WidgetProject'
+
+const { localVue, config, i18n, store } = Core.init(createLocalVue()).useAll()
+
+describe('WidgetProject.vue', () => {
```

Comment:
I like how the tests are written it is pleasant to read

Reply:
Thanks!